### PR TITLE
 TRUNK-5712:

### PR DESCRIPTION
**Pull request by mumbereh**
## Issue I worked on

see https://issues.openmrs.org/browse/TRUNK-

## Checklist: I completed these to help reviewers :)

- [ ] My IDE is configured to follow the [**code style**](https://wiki.openmrs.org/display/docs/Java+Conventions) of this project.

  No? Unsure? -> [configure your IDE](https://wiki.openmrs.org/display/docs/How-To+Setup+And+Use+Your+IDE), format the code and add the changes with `git add . && git commit --amend`

- [ ] I have **added tests** to cover my changes. (If you refactored
  existing code that was well tested you do not have to add tests)

  No? -> write tests and add them to this commit `git add . && git commit --amend`

- [ ] I ran `mvn clean package` right before creating this pull request and
  added all formatting changes to my commit.

  No? -> execute above command

- [ ] All new and existing **tests passed**.

  No? -> figure out why and add the fix to your commit. It is your responsibility to make sure your code works.

- [ ] My pull request is **based on the latest changes** of the master branch.

  No? Unsure? -> execute command `git pull --rebase upstream master`

### CODE REVIEW

**Comment by tendomart** [comment]
What is this PR for ?

Meanwhile you want to work on this

1. As a rule of the thumb always provide Title and link to the Ticket.
2. Remove all wild cards from Imports

File: api/src/main/java/org/openmrs/module/ModuleFactory.java [hunk 1]
```diff
@@ -9,12 +9,12 @@
...
 package org.openmrs.module;
 
-import java.io.File;
+import java.io.File; 
 import java.io.IOException;
 import java.io.InputStream;
 import java.net.MalformedURLException;
 import java.net.URL;
-import java.rmi.activation.Activator;
+import java.rmi.*;
```
Comment (tendomart) [inline api/src/main/java/org/openmrs/module/ModuleFactory.java:17]: Remove this wildcard import. The path should be absolute

Comment (tendomart) [reply api/src/main/java/org/openmrs/module/ModuleFactory.java]: @mumbereh  have you looked at my review comments above ?

**Comment by tendomart** [comment]
Did you act upon all the comments ?

any way look at this.

https://wiki.openmrs.org/display/docs/Pull+Request+Tips

**Comment by jwnasambu** [comment]
@mumbereh your ticket Id should look like this: `TRUNK-5712:javascript MIME Type`

**Pull request closed without merging**